### PR TITLE
Created a pipeline file

### PR DESCRIPTION
**Pull request by CharliedeanCS**
Created a Pipeline file that extracts,transforms and loads data every minute from the API and uploads it to the database.

All functions as intended
Code runs every minute 
Code is commented to clearly state what happens at each stage

### CODE REVIEW

File: pipeline/pipeline.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+"""
+Pipeline script that connects the ETL scripts,
+Runs the pipeline in a loop every 1 min.
+"""
+
+import time
```
Comment (shivani-63) [inline pipeline/pipeline.py:6]: Really nice imports, well formatted and nice clear docstring!

File: pipeline/pipeline.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+from extract import fetch_all_plant_data, convert_plant_data_to_csv
+from transform import standardize_country_name, remove_rows_with_null, check_soil_moisture_valid, check_soil_temp_valid, normalize_datetimes, change_temp_and_moisture_to_two_dp
+from load import create_database_connection, insert_into_recordings_table
+
+if __name__ == "__main__":
+
+    load_dotenv()
+
+    while True:
```
Comment (shivani-63) [inline pipeline/pipeline.py:20]: Do you think it's worth putting this chunk of code in a function, then just calling the function in the if __name__ == "__main__" block? Either way, it's clear and makes sense , but putting it in a function might be slightly better stylistically?

File: pipeline/pipeline.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+        # Starts a timer for each iteration of the pipeline
+        st = time.time()
+
+        # Fetches all plant data from the api
+        plant_api_data = fetch_all_plant_data()
+
+        # Creates a csv file for the data and returns a data frame
+        plants = convert_plant_data_to_csv(plant_api_data)
```
Comment (zandersnow14) [inline pipeline/pipeline.py]: Will this create a CSV file every time the loop runs? Don't think we need this - maybe just replace this with `plants = pd.Dataframe(plant_api_data)`?

File: pipeline/pipeline.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+import time
+from os import environ
+
+from dotenv import load_dotenv
+
+from extract import fetch_all_plant_data, convert_plant_data_to_csv
+from transform import standardize_country_name, remove_rows_with_null, check_soil_moisture_valid, check_soil_temp_valid, normalize_datetimes, change_temp_and_moisture_to_two_dp
+from load import create_database_connection, insert_into_recordings_table
+
+if __name__ == "__main__":
```
Comment (zandersnow14) [inline pipeline/pipeline.py:16]: Like Shivani said, might be good to split up this into functions - maybe have extract, transform and load functions?

**Review by shivani-63 — APPROVED** [review]

**Pull request merged**